src/container/BooksList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BooksList = ({ bookId, title, category }) => (
+  <tr>
+    <th>BookId</th>
+    <th>Title</th>
+    <th>Category</th>
+    <Book bookId={bookId} title={title} category={category} />
+  </tr>
+);
```

Comment:
- [ ] <b>REQUIRED:</b> `Booklist` should receive the list of the books not just single book from props. and it should display those books in a table using `Book` component. Kindly fix that. [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/BooksAsTable.png)
Here is an example:
```Javascript
const BooksList = (books) => (
  <tr>
    <th>BookId</th>
    <th>Title</th>
    <th>Category</th>
    {
      books.map((book) => (<Book bookId={book.bookId} title={book.title} category={book.category} />))
    }
  </tr>
);
```